src/assets/styles/main.css

```diff
@@ -4,18 +4,34 @@
   margin: 0;
 }
 
-html {
-  font-size: 0.625rem;
```

Comment:
->

Reply:
I'll see what causes the relative property to not work.
I'll also change my code to work with .625rem